src/applications/facility-locator/components/SearchResult.jsx

```diff
@@ -3,7 +3,32 @@ import PropTypes from 'prop-types';
 import LocationInfoBlock from './search-results/LocationInfoBlock';
 import LocationPhoneLink from './search-results/LocationPhoneLink';
 import LocationDirectionsLink from './search-results/LocationDirectionsLink';
+import { CLINIC_URGENTCARE_SERVICE, LocationType } from '../constants';
 
+export const urgentCareCall = query => {
+  const content = () => (
+    <p>
+      {' '}
+      Before going to a clinic for urgent care, please call the facility to
+      confirm that it's open and able to provide the care you need.{' '}
+    </p>
+  );
+
+  if (
+    query.facilityType === LocationType.URGENT_CARE &&
+    query.serviceType === 'NonVAUrgentCare'
+  ) {
+    return content();
+  }
+  if (
+    query.facilityType === LocationType.CC_PROVIDER &&
+    query.serviceType === CLINIC_URGENTCARE_SERVICE
+  ) {
+    return content();
```

Comment:
`content` doesn't need to be a function (could just be a chunk of JSX) but it's up to you!